flashchips.c

```diff
@@ -9573,6 +9573,34 @@ const struct flashchip flashchips[] = {
 		.voltage	= {2700, 3600},
 	},
 
+
+	{
+		.vendor		= "Microchip",
+		.name		= "25LC640",
+		.bustype	= BUS_SPI,
+		.manufacture_id	= 0, /* Not used. */
+		.model_id	= 0,
+		.total_size	= 8,
+		.page_size	= 32,
+		.feature_bits	= FEATURE_2BA | FEATURE_NO_ERASE,
+		.tested		= TEST_OK_PREW,
+		.probe		= NULL,
+		.probe_timing	= TIMING_ZERO,
+		.block_erasers	=
+		{
+			{
+				.eraseblocks = { {8192, 1} },
+				.block_erase = spi_block_erase_emulation,
+			}
+		},
+		.printlock	= spi_prettyprint_status_register_bp1_srwd, /* TODO: check */
+		.unlock		= spi_disable_blockprotect_bp1_srwd,
+		.write		= spi_chip_write_256,
+		.read		= spi_chip_read,
+		.voltage	= {5000, 5000},
```

Comment:
Hello,

By looking at the datasheet I see different range of the voltage:
![kép](https://user-images.githubusercontent.com/1609182/88475116-5ed95200-cf2d-11ea-8ff4-bbb78d91838e.png)

There are flashrom supported programmers which supports adjustable IO voltage so it is useful to keep these values correct.